pkg/plugins/runner/runner_test.go

```diff
@@ -0,0 +1,34 @@
+package runner
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/nornir-automation/gornir/pkg/gornir"
+)
+
+var (
+	testHosts = map[string]*gornir.Host{
+		"dev1": {Hostname: "dev1"},
+		"dev2": {Hostname: "dev2"},
+		"dev3": {Hostname: "dev3"},
+		"dev4": {Hostname: "dev4"},
+	}
+)
+
+type testTaskSleep struct {
+	sleepDuration time.Duration
+}
+
+type testTaskSleepResults struct {
+	success bool
+}
+
+func (t *testTaskSleep) Run(ctx context.Context, wg *sync.WaitGroup, jp *gornir.JobParameters, jobResult chan *gornir.JobResult) {
```

Comment:
This is just a personal preference, but I try to keep the name of the inputs short, so `jobResult` could be `jr` or just `ch`, then the line `jobResult <- result` would be something like; `ch <- result`.

```go
func (t *testTaskSleep) Run(ctx context.Context, wg *sync.WaitGroup, jp *gornir.JobParameters, jobResult chan *gornir.JobResult)
```

to

 ```go
func (t *testTaskSleep) Run(ctx context.Context, wg *sync.WaitGroup, jp *gornir.JobParameters, jr chan *gornir.JobResult)
```

Reply:
Only reason to keep it long was that I didn't know how long tasks could be some something descriptive felt better. For instance, with `jp` you have to type cast it probably in the first few lines so you can name it in the new var however you want but `jobResult` might be consumed 15-20 lines after. Anyway, I think as long as we are consistent with the name it's not going to be a problem. Will look into changing it everywhere.

Reply:
Look at the ssh task after the change and let me know what you think.